src/components/content/catalog/services/update/UpdateService.tsx

```diff
@@ -267,7 +276,31 @@ function UpdateService({
                             </Row>
                         </Upload>
                     </div>
-                    <div>{oclDisplayData.current}</div>
+                    <div>
+                        {files.current.length > 0 ? (
+                            <div className={registerStyles.registerButtons}>
+                                <Row>
+                                    <Form.Item
+                                        name='isRemoveServiceTemplateUntilApproved'
+                                        label={
+                                            <span style={{ fontWeight: 'bold' }}>
+                                                RemoveServiceTemplateUntilApproved
```

Comment:
Better to call the name - 'Remove current service template until updated service template is approved'